depsync/depsync.go

```diff
@@ -159,18 +233,26 @@ FETCH:
 		} else if err != nil {
 			return err
 		}
-		fmt.Printf("more=%v, cursor=%s, fetched=%v\n", resp.MoreToFollow, resp.Cursor, resp.FetchedUntil)
+		level.Info(w.logger).Log("msg", "DEP fetch", "more", resp.MoreToFollow, "cursor", resp.Cursor, "fetched", resp.FetchedUntil, "devices", len(resp.Devices))
```

Comment:
use newlines to make this more readable. 

```
level.Info(w.logger).Log(
"msg", "DEP fetch",
"more", resp.MoreToFollow,
)
```
 keep at key, value per line

Reply:
👍